vita3k/kernel/src/thread/thread.cpp

```diff
@@ -102,6 +102,17 @@ SceUID create_thread(Ptr<const void> entry_point, KernelState &kernel, MemState
         return SCE_KERNEL_ERROR_ERROR;
     }
 
+    thread->cpu_context = init_cpu_context();
```

Comment:
```suggestion
    thread->cpu_context = std::make_unique<CPUContext>();
```
`#include <memory>` too